app/assets/tailwind/application.css

```diff
@@ -913,3 +913,26 @@ input[type="checkbox"] {
 input[type="radio"] {
   @apply text-primary-600 checked:border-primary-500 dark:checked:border-primary-500 dark:checked:bg-primary-500 mt-0.5 shrink-0 rounded-full border-slate-200 disabled:pointer-events-none disabled:opacity-50 dark:border-slate-700 dark:bg-slate-600;
 }
+
```

Comment:
Can you see if there is a way for us to change the color of the icon (background-image) in dark mode? Currently looks like this:

![image](https://github.com/user-attachments/assets/0656e618-f7aa-483b-882c-1d2f402a2290)